docs/usage/ippool-multi.md

```diff
@@ -1,21 +1,13 @@
-# Multiple IPPool
+# Backup IPPool
 
-Ippool contains ranges of IP address belonging to a subnet. It has some usages:
-
-* multiple application not requiring static IP could share a same ippool.
-
-* Multiple IP pools can be set for a pod for the usage of backup IP resources.
+Multiple IP pools can be set for a pod for the usage of backup IP resources.
 
 ## Get Started
 
 ### Set up Spiderpool
 
 follow the guide [installation](https://github.com/spidernet-io/spiderpool/blob/main/docs/usage/install.md) to install Spiderpool.
 
-### applications share a same ippool
-
-TODO
-
 ### backup ippool effect
 
 create two IPPools each containing 2 IP addresses.
```

Comment:
Based on the provided code patch, there doesn't seem to be any actual code changes. It appears to be mainly updates to the comments and documentation.

As for the comments themselves, they appear to be clear and informative. However, one improvement suggestion would be to provide examples or further explanations for some of the concepts mentioned, such as how multiple IP pools can be used for backup resources. Additionally, the "TODO" comment regarding applications sharing a same ippool should be completed or removed.

Without seeing the actual code implementation, it is difficult to identify any bug risks.